code/object/objcollide.cpp

```diff
@@ -44,7 +44,7 @@ class collider_pair
 	{}
 };
 
-static bool Collision_cache_stale = false;
+static SCP_vector<object*> Collision_cache_stale_objects;
```

Comment:
I wonder if this should be a set rather than a vector.  It looks like the vector will still work, but it might store more than one copy of the same pointer if the same object was invalidated more than once in the same frame.

Reply:
Ah, I didn't know C++ had sets! Good point